### ISSUE TITLE
Sometimes photos don't increment

### ISSUE THREAD

**Issue by nickoneill** [5calls/5calls#363]
In this case I was flipping through net neutrality, so doing Pai -> Feinstein -> Harris -> Pelosi

And Harris was showing Feinstein. Everything else changed correctly. Maybe Harris doesn't have a photo and it doesn't update if there's no new photo contact?

**Comment by rupl** [comment]
It feels like the image load is just delaying the swap.

I've noticed a delay when progressing through each person, but it usually ends up loading the correct photo after a moment. I also saw this while calling about net neutrality, but it also happened for another issue while progressing through my local senators. The photo loaded after less than 5 seconds delay, but for a brief moment it was confusing.

**Comment by nickoneill** [comment]
Agreed, regardless of if Harris has a photo or not, the lack of unloading the image is jarring. Are you interested in taking a look at this @rupl?

We could either unset the image src or use the default image that we've used for offices/committees in the past when there is no profile image: https://5calls.org/img/5calls-icon-office.png

**Comment by rupl** [comment]
Sure I can take a look, although my react-fu is pretty weak.

I did take a peek at my DevTools Network tab while making today's round of calls and in my case, Sen Cruz has a huuuge 1MB image compared to Sen Cornyn's 38KB, and Cruz always comes after Cornyn so this is happening regularly for me. The image download does seem to be the bottleneck.

I imagine React probably has some mechanism allowing devs to specify a preload for the next asset the visitor will need. I have no idea how to accomplish that, but that's an alternative to loading a generic image.

**Comment by nickoneill** [comment]
Ah, we're getting the suggested image url from google civic, they must be pointing at something giant :(

Alternatively, if your forte is closer to aws lambda and image manipulation, we could use a better system for requesting/caching correctly sized images.

Something like:
`api.5calls.org/image/<some open civic data id>` -> look at s3 for an image, return 301 redirect if it exists -> if not, find image from some civic api, download and resize and store on s3 -> return image
